server/lib/nl/page_config_builder.py

```diff
@@ -100,6 +100,7 @@ def build_page_config(
     all_svs.update(cspec.svs)
   all_svs = list(all_svs)
   sv2name = utils.get_sv_name(all_svs)
+  sv2unit = utils.get_sv_unit(all_svs)
```

Comment:
Add a TODO to extend to map and ranking too?

Reply:
done